.github/workflows/integrate-v3.yml

```diff
@@ -9,6 +9,7 @@ on:
 env:
   SLS_IGNORE_WARNING: '*'
   FORCE_COLOR: 1
+  SLS_SCHEMA_CACHE_BASE_DIR: '/home/runner'
```

Comment:
good call, changed

Reply:
I would also remove changes in -v3` workflow files from this commit, and add them to first commit in v3 branch (this workflow files will never be added in master, so that change will be conflicting).

But we do such rebase after merging this to v3

Reply:
Good suggestion, so let's merge this one and edit the original v3 commit as a part of rebase afterwards, right?

Reply:
Yes, exactly :+